lasagne/updates.py

```diff
@@ -598,6 +598,65 @@ def adam(loss_or_grads, params, learning_rate=0.001, beta1=0.9,
     return updates
 
 
+def adamax(loss_or_grads, params, learning_rate=0.002, beta1=0.9,
+         beta2=0.999, epsilon=1e-8):
```

Comment:
Maybe the algo is already in downhill that use Theano. This could be used
in some cases:

http://downhill.readthedocs.org/en/stable/

On Wed, Dec 16, 2015 at 9:29 AM, Jan Schlüter <notifications@github.com>
wrote:

> In lasagne/updates.py
> <https://github.com/Lasagne/Lasagne/pull/540#discussion_r47781316>:
>
> > @@ -598,6 +598,65 @@ def adam(loss_or_grads, params, learning_rate=0.001, beta1=0.9,
> >      return updates
> >
> >
> > +def adamax(loss_or_grads, params, learning_rate=0.002, beta1=0.9,
> > +         beta2=0.999, epsilon=1e-8):
>
> pep8: two more blanks needed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Lasagne/Lasagne/pull/540/files#r47781316>.
>